src/QMCDrivers/DMC/WalkerControlFactory.cpp

```diff
@@ -41,7 +41,10 @@ WalkerControlBase* createWalkerController(int nwtot, Communicate* comm, xmlNodeP
   //if(nmin<0) nmin=nideal/2;
   WalkerControlBase* wc = 0;
   int ncontexts         = comm->size();
-  bool fixw             = (reconfig || reconfigopt == "yes" || reconfigopt == "pure");
+  if((reconfig_str != "no" && reconfig_str != "runwhileincorrect")
+    APP_ABORT("Reconfiguration is currently broken.  Please try again without reconfiguration.")
+  //bool fixw             = (reconfig || reconfigopt == "yes" || reconfigopt == "pure");
```

Comment:
Please replace the commented line with a commend saying "pure" was a valid input to be restored in the future.

Reply:
"Reconfiguration variants of DMC are currently broken".

This is getting ahead of ourselves, but what we reenable in future can be determined in future.  Since we have never had a proper verification test of "pure" I consider it just as suspect as reconfiguration. The manual also needs to contain an exact definition of whatever methods are implemented. In my opinion we only need one efficient fixed population method to work and be implemented.

Reply:
Was "pure" ever a valid input?  It only appears here (try to grep on "pure") and has the same effect as "yes".

Reply:
When I left the comment, I thought "pure" is DMC without branching.
It seems that we don't have any related code path. So just remove it.
We probably should have it added eventually and re-enable an input entry.

Reply:
It is mentioned in the manual, so perhaps someone thought it was or there is an old input with pure mentioned. Provided the abort message is clear (put no or remove the option), aborting for !="no" is fine by me. 

(It would be far more appropriate to have an input option called populationcontrol handling these different methods... however this is getting far ahead of ourselves since no one has been able to identify the actual bug in the reconfiguration implementation...)

Reply:
Since eventually "no" is the only option left, we can remove this old option and introduce a new option once reconfiguration is fixed.

Reply:
I agree.